llvm/docs/ReleaseNotes.md

```diff
@@ -449,6 +449,8 @@ Changes to LLDB
           _regexp-bt        -- Show backtrace of the current thread's call ...
           _regexp-display   -- Evaluate an expression at every stop (see 'h...
   ```
+* DWARF indexing speed (for binaries not using the debug_names index) increased
+  by 30-60%, depending on the configuration.
```

Comment:
Configuration of what, perhaps it is of the compiler producing the binary, which changes the complexity of the debug info?

Reply:
Here I mainly meant the type of dwarf being used (type units, dwo vs. dwp, etc.), though I haven't done some comprehensive tests to know what are all the variables. I don't know how much detail do we want to go in to here. I could change this to "depending on the kind debug info used (type units or not, dwo vs. dwp vs. regular DWARF, ...)", or just omit the part after the comma completely. (I could also omit the entry altogether, but I though people might like to see that)

Reply:
I imagine a reader who thinks this is cool but wonders if they can get even more of a speedup. There are probably < 5 people and 3 of them are you and your reviewers but who knows.

To keep it brief, you could remove the bit after the comma and link to the PRs instead, or to the single, key PR. They have all the detail anyone could possibly want.

Reply:
Done (after some wrestling with the github "edit file" UI).